.circleci/config.yml

```diff
@@ -956,6 +956,38 @@ workflows:
             - attach_workspace:
                 at: /
 
+  GCP-Daily-Services-Comp-Update-Reconnect-Abort-4N-2C:
+    jobs:
+      - build-platform-and-services
+      - jrs-regression:
+          context: Slack
+          regression_path: /swirlds-platform/regression
+          result_path: results/4N_2C/UpdateReconnectAbort
+          config_type: "daily"
+          workflow-name: "GCP-Daily-Services-Comp-Update-Reconnect-Abort-4N-2C"
+          requires:
+            - build-platform-and-services
+          pre-steps:
+            - install-tools
+            - attach_workspace:
+                at: /
+
+  GCP-Daily-Services-Comp-Update-Reconnect-Abort-DisPreUpdate-4N-2C:
+    jobs:
```

Comment:
need to add as a scheduled cron job , f.e
```
- schedule:
          cron: "30 6 * * *"
          filters:
            branches:
              only:
                - master

Reply:
scheduled added